src/views/UserSearch/Filters.js

```diff
@@ -4,24 +4,24 @@ import {
   FormattedMessage,
   injectIntl,
 } from 'react-intl';
-import {
-  get,
-} from 'lodash';
+import { get } from 'lodash';
 
 import {
   Accordion,
   AccordionSet,
   FilterAccordionHeader,
 } from '@folio/stripes/components';
-
+import { stripesConnect } from '@folio/stripes/core';
 import {
   CheckboxFilter,
   MultiSelectionFilter,
 } from '@folio/stripes/smart-components';
 
-import { statusFilter } from '../../constants';
-
 import CustomFieldsFilters from '../../components/CustomFieldsFilters';
+import { isConsortiumEnabled } from '../../components/util';
+import { USER_TYPES, statusFilter } from '../../constants';
+
+const ACCORDION_ID_PREFIX = 'users_filter_accordion';
```

Comment:
Constants, ❤️ . But please keep the same value that was there previously, with hyphens instead of underscores. I know this didn't break any tests, but who knows what else might rely on that value; if we can preserve it then we should.